src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -52,8 +45,6 @@ public class SnowflakeSinkServiceV2 implements SnowflakeSinkService {
 
   private static final KCLogger LOGGER = new KCLogger(SnowflakeSinkServiceV2.class.getName());
 
-  private static String STREAMING_CLIENT_PREFIX_NAME = "KC_CLIENT_";
```

Comment:
most of this logic was moved to the StreamingClientHandler.java file, which was merged in this last PR:
https://github.com/snowflakedb/snowflake-kafka-connector/pull/590/files